templates/todo/projects/nodejs-mongo-aca/.repo/bicep/azure.yaml

```diff
@@ -28,7 +39,7 @@ hooks:
   predeploy:
     windows:
       shell: pwsh
-      run: 'echo "VITE_API_BASE_URL=\"$env:API_BASE_URL\"" > ./src/web/.env.local ; echo "VITE_APPLICATIONINSIGHTS_CONNECTION_STRING=\"$env:APPLICATIONINSIGHTS_CONNECTION_STRING\"" >> ./src/web/.env.local'
+      run: 'echo "VITE_API_BASE_URL=$env:API_BASE_URL" > ./src/web/.env.local ; echo "VITE_APPLICATIONINSIGHTS_CONNECTION_STRING=$env:APPLICATIONINSIGHTS_CONNECTION_STRING" >> ./src/web/.env.local'
```

Comment:
I'm using windows 11 with PowerShell 7.4.2 (`pwsh --version` shows PowerShell 7.4.2) and it causes the .env.local to look like this when running the `echo` command:
```
VITE_API_BASE_URL=/
https://ca-api-zdntfchj4zeri.thankfulground-ec33f819.eastus.azurecontainerapps.io/
VITE_APPLICATIONINSIGHTS_CONNECTION_STRING=/
InstrumentationKey=cb775b3f-ca95-4b0d-9b4c-02d386bd39f9;IngestionEndpoint=https://eastus-8.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/;ApplicationId=38cb1ed5-58f0-4952-8f19-ce6b20597d9b/
```
So the bundled web container can't read connection appinsight string correctly and the web application in browser stuck at loading

Reply:
@vhvb1989 - Thoughts on this need?

Reply:
this was fixed here: https://github.com/Azure/azure-dev/pull/3671/files

You might want to pull upstream

Reply:
fixed by merging the hook fix.